README.md

```diff
@@ -137,9 +145,6 @@ with plt.style.context("cosmoplots.default"):
 | :--------: | :--------: |
 | ![matplotlib](./assets/matplotlib.png) | ![cosmoplots](./assets/cosmoplots.png) |
 
-<!-- Links -->
-[poetry]: https://python-poetry.org
```

Comment:
This breaks the link to Poetry further up in the README.

Reply:
Should work now.